validator/client/log.go

```diff
@@ -71,9 +74,10 @@ func (v *validator) LogNextDutyTimeLeft(slot uint64) error {
 		if uint64(timeLeft) >= params.BeaconConfig().SecondsPerSlot {
 			log.WithFields(
 				logrus.Fields{
-					"role":        role,
 					"currentSlot": slot,
 					"dutySlot":    nextDutySlot,
+					"attesting":   attestingCounts[nextDutySlot],
```

Comment:
This will be 0 if `nextDutySlot` doesn't exist.  I dont think an existence check to return error is particularly necessary